app/routes/poll-discourse.tsx

```diff
@@ -0,0 +1,97 @@
+import { useFetcher, useLoaderData } from "@remix-run/react"
+import { useEffect, useState } from "react"
+import {
+  fetchBreakingChangesPosts,
+  fetchMainnetSporkPosts,
+  Topic,
+} from "~/cms/utils/fetch-discourse-api"
+import { formatDistance } from "date-fns"
+import { POLLING_INTERVAL } from "~/cms/utils/constants"
+
+type LoaderData = {
+  breakingChanges: Topic[]
+  sporks: Topic[]
+}
+
+const ABOUT_THIS_CATEGORY_BREAKING_CHANGES = 762
+const ABOUT_THIS_CATEGORY_SPORK = 2543
+
+function sortTopics(topics: Topic[], removeCategoryId: number) {
+  const sorted: Topic[] = topics
+    .filter((post: Topic) => post.id !== removeCategoryId)
+    .map((post: Topic) => {
+      const date = formatDistance(new Date(post.created_at), new Date())
+      post.__formatted_date = date
```

Comment:
@codingone21 sorry i didn't look at this closely, formatting the date IMO should happen in the react component, e..g a component for this:


```tsx
<li id="user-content-fn-1" key={t.id}>
            Breaking Change Topic: <RelativeDate={t.created_at} /> - {t.title}
          </li>
```

or a function taht returns a string

```tsx
<li id="user-content-fn-1" key={t.id}>
            Breaking Change Topic: {relativeDate(t.created_at)} - {t.title}
          </li>
```

Reply:
Yep, I realized this while reviewing my changes too! My bad, will update this to the correct approach